### DISCUSSION TITLE
v2.1.5

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3039] [分享 / Share]
✨**正式版** `v2.1.5`

- 修复选项出现 null 导致脚本无法运行. (#2928)

----

✨**预览版** `v2.1.5-preview`

- 修复搜索框中的 `检查更新` 和设置中的 `立即检查所有更新` 在未超过更新间隔期时没有运行.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.5'>v2.1.5</a>.</em>

**Comment by fly-pigeon** [comment]
问一下，为啥我下载了这个zip，导入油猴里面变成2.1.4版本了啊
上次导入2.1.4版本的时候显示的是2.1.3

**Reply by the1812** [reply]
去首页链接安装...
